USBPcapCMD/cmd.c

```diff
@@ -1232,16 +1236,17 @@ int __cdecl main(int argc, CHAR **argv)
         /* Extcap interface. Please note that there are no short
          * options for these and the numbers are just gopt keys.
          */
-        {'1', 0, "", _1_long},
-        {'2', GOPT_ARG, "", _2_long},
-        {'3', 0, "", _3_long},
+        {'1', GOPT_ARG, "", _1_long},
```

Comment:
I think GOPT_ARG | GOPT_REPEAT would be appropriate here.

Atleast this is what I think the
"The version for the extcap sentence (which may exist as many times as is needed, but only the last one will be used) will be used for displaying the version information of the extcap interface in the about dialog of Wireshark." in https://github.com/wireshark/wireshark/blob/master/docbook/wsdg_src/WSDG_chapter_capture.asciidoc means. If I am wrong, could you please correct me?

Reply:
Nope, that is about the _response_ the extcap program sends back to Wireshark. The command line parameter extcap-version, used when calling the extcap program, only appears once.